pool/routes/routes.ml

```diff
@@ -116,7 +123,7 @@ module Public = struct
               [ CustomMiddleware.Guardian.require_user_type_of
                   Pool_context.UserType.[ Contact; Admin ]
               ]
-            [ get "/logout" Login.logout ]
+            [ get "/logout" Login.logout; announcements ]
```

Comment:
Should announcements be public?

Reply:
No, the issue states that the banner should be shown to contacts and/or admins